ui/web-v2/Makefile

```diff
@@ -1,3 +1,6 @@
+.PHONY: all
+all: install buildifier-check lint gen_proto build
```

Comment:
I added this so people can check everything locally before opening the PR.

Reply:
Thank you! 😄